api/repo/update.go

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) 2022 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package repo
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-vela/server/database"
+	"github.com/go-vela/server/router/middleware/org"
+	"github.com/go-vela/server/router/middleware/repo"
+	"github.com/go-vela/server/router/middleware/user"
+	"github.com/go-vela/server/util"
+	"github.com/go-vela/types/constants"
+	"github.com/go-vela/types/library"
+	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
+)
+
+// swagger:operation PUT /api/v1/repos/{org}/{repo} repos UpdateRepo
+//
+// Update a repo in the configured backend
+//
+// ---
+// produces:
+// - application/json
+// parameters:
+// - in: path
+//   name: org
+//   description: Name of the org
+//   required: true
+//   type: string
+// - in: path
+//   name: repo
+//   description: Name of the repo
+//   required: true
+//   type: string
+// - in: body
+//   name: body
+//   description: Payload containing the repo to update
+//   required: true
+//   schema:
+//     "$ref": "#/definitions/Repo"
+// security:
+//   - ApiKeyAuth: []
+// responses:
+//   '200':
+//     description: Successfully updated the repo
+//     schema:
+//       "$ref": "#/definitions/Repo"
+//   '400':
+//     description: Unable to update the repo
+//     schema:
+//       "$ref": "#/definitions/Error"
+//   '500':
+//     description: Unable to update the repo
+//     schema:
+//       "$ref": "#/definitions/Error"
+//   '503':
+//     description: Unable to update the repo
+//     schema:
+//       "$ref": "#/definitions/Error"
+
+// UpdateRepo represents the API handler to update
+// a repo in the configured backend.
+//
+//nolint:funlen // ignore line length
```

Comment:
This is just a copy/paste from where it was before.

It might be nice to fix this comment however, as it doesn't seem right. This is ignoring the function length, not the line length, isn't it?

Reply:
fixed 👍